packages/notification-center/src/components/notification-center-component.tsx

```diff
@@ -0,0 +1,203 @@
+import React, { FunctionComponent } from 'react';
+import { IMessage, MessageActionStatusEnum, ButtonTypeEnum } from '@novu/shared';
+
+import { INovuProviderProps, NovuProvider } from './novu-provider';
+import { IPopoverNotificationCenterProps, PopoverNotificationCenter } from './popover-notification-center';
+import { NotificationBell } from './notification-bell';
+import { useNotifications } from '../hooks';
+import { reactToWebComponent } from '../utils';
+import { ColorScheme } from '../shared/config/colors';
+
+type NovuProviderProps = Pick<
+  INovuProviderProps,
+  | 'onLoad'
+  | 'stores'
+  | 'backendUrl'
+  | 'socketUrl'
+  | 'subscriberId'
+  | 'applicationIdentifier'
+  | 'subscriberHash'
+  | 'i18n'
+> & {
+  // Angular props
+  sessionLoaded?: INovuProviderProps['onLoad'];
+};
+type PopoverProps = Pick<
+  IPopoverNotificationCenterProps,
+  | 'onUrlChange'
+  | 'onUnseenCountChanged'
+  | 'onActionClick'
+  | 'onTabClick'
+  | 'theme'
+  | 'tabs'
+  | 'showUserPreferences'
+  | 'offset'
+  | 'position'
+> & {
+  onNotificationClick?: IPopoverNotificationCenterProps['onNotificationClick'];
+  colorScheme?: ColorScheme;
+  // Angular props
+  urlChanged?: IPopoverNotificationCenterProps['onUrlChange'];
+  notificationClicked?: IPopoverNotificationCenterProps['onNotificationClick'];
+  unseenCountChanged?: IPopoverNotificationCenterProps['onUnseenCountChanged'];
+  actionClicked?: IPopoverNotificationCenterProps['onActionClick'];
+  tabClicked?: IPopoverNotificationCenterProps['onTabClick'];
+};
+type BellProps = { unseenBadgeColor?: string; unseenBadgeBackgroundColor?: string };
+
+export type NotificationCenterComponentProps = NovuProviderProps & PopoverProps & BellProps;
+
+type PopoverWrapperProps = PopoverProps & BellProps;
+
+/*
+ * This array represents the public API of the web component.
+ * All the props defined in the NotificationCenterComponentProps should be added here.
+ */
+export const NOTIFICATION_CENTER_PROPS = [
+  // NovuProvider props
+  'onLoad',
+  'sessionLoaded',
+  'stores',
+  'backendUrl',
+  'socketUrl',
+  'subscriberId',
+  'applicationIdentifier',
+  'subscriberHash',
+  'i18n',
+  // PopoverNotificationCenter props
+  'onUrlChange',
+  'onNotificationClick',
+  'onUnseenCountChanged',
+  'onActionClick',
+  'onTabClick',
+  'colorScheme',
+  'theme',
+  'tabs',
+  'showUserPreferences',
+  'offset',
+  'position',
+  // NotificationBell props
+  'unseenBadgeColor',
+  'unseenBadgeBackgroundColor',
+];
```

Comment:
this array is a list of props the `react-to-webcomponent` library is using and listening to on the proxy component